lib/Pithub/Repos.pm

```diff
@@ -85,6 +85,79 @@ sub branches {
     );
 }
 
+=method rename_branch
+
+=over
+
+=item *
+
+Rename a branch
+
+    POST /repos/:user/:repo/branches/:branch/rename
+
+Examples:
+
+    my $b = Pithub::Repos->new;
+    my $result = $b->rename_branch(
+        user => 'plu',
+        repo => 'Pithub',
+        branch  => 'travis',
+        data => { new_name => 'travis-ci' }
+    );
+
+=back
+
+=cut
+
+sub rename_branch {
+    my ( $self, %args ) = @_;
+    croak 'Missing parameters: data' unless $args{data};
+    croak 'Missing parameters: branch' unless $args{branch};
+    croak 'Missing key in parameters: new_name' unless $args{data}->{new_name};
```

Comment:
Could you double check the `croak` messages here to make sure they're in line with the rest of the API?

Reply:
I aligned with the rest of API :+1: